drivers/spi/spi_esp32_spim.c

```diff
@@ -505,6 +517,7 @@ static const struct spi_driver_api spi_api = {
 		.dma_host = DT_INST_PROP(idx, dma_host),	\
 		.cs_setup = DT_INST_PROP_OR(idx, cs_setup_time, 0), \
 		.cs_hold = DT_INST_PROP_OR(idx, cs_hold_time, 0), \
+		.line_idle_low = DT_INST_PROP_OR(idx, line_idle_low, 0), \
```

Comment:
Boolean DTS properties will always be defined. 
```suggestion
		.line_idle_low = DT_INST_PROP(idx, line_idle_low), \
```